docs/getting-started/bpfctl-guide.md

```diff
@@ -115,18 +112,24 @@ Options:
           the byte string appropriately considering such things as size, endianness,
           alignment and packing of data structures.
 
-      --map-owner-uuid <MAP_OWNER_UUID>
-          Optional: UUID of loaded eBPF program this eBPF program will share a map with.
-          Only used when multiple eBPF programs need to share a map. If a map is being
-          shared with another eBPF program, the eBPF program that created the map can not
-          be unloaded until all eBPF programs referencing the map are unloaded.
-          Example: --map-owner-uuid 989958a5-b47b-47a5-8b4c-b5962292437d
+  -m, --metadata <METADATA>
+          Optional: Specify Key/Value metadata to be attached to a program when it
+          is loaded by bpfd.
+          Format: <KEY>=<VALUE>
+          
+          This can later be used to list a certain subset of programs which contain
+          the specified metadata.
+
+      --map-owner-id <MAP_OWNER_ID>
+          Optional: Program id of loaded eBPF program this eBPF program will share a map with.
+          Only used when multiple eBPF programs need to share a map.
+          Example: --map-owner-id 63178
 
   -h, --help
           Print help (see a summary with '-h')
 ```
 
-So when using `bpfctl load-from-file`, `--path`, `--section-name`, `--id`, `--global`
+When using `bpfctl load-from-file`, `--path`, `--section-name`, `--id`, `--global`
```

Comment:
```suggestion
When using `bpfctl load-from-file`, `--path`, `name`, `--id`, `--global`
```

Reply:
We atually need to do a scrub of all the cli commands and output in the docs since things have changed.